server.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express')
+const path = require('path')
+const app = express()
+
+const port = process.env.PORT || 5000
+const staticDir=process.env.DEV ? './client/public' : "./client/build"
```

Comment:
This line currently isn't being used for this server. We can remove it and your server will run the same way